app/src/main/res/layout/content_item.xml

```diff
@@ -48,7 +48,7 @@
       android:breakStrategy="simple"
       android:fontFamily="sans-serif"
       android:text="@{htmlContent}"
-      android:textColor="@color/oppia_primary_text"
+      android:textColor="@color/question_player_activity_textview_highlighted_color"
```

Comment:
Considering this color might be getting used at a lot of places, I suggest you name it as follows:

`shared_player_text_view_primary_color`

Reply:
Or if you want to create specific colors then name it as `content_item_text_view_color`

Reply:
To me first one looks much better